ipaserver/install/httpinstance.py

```diff
@@ -263,6 +265,19 @@ def enable_mod_nss_renegotiate(self):
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRenegotiation', 'on', False)
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRequireSafeNegotiation', 'on', False)
 
+    def enable_mod_nss_ocsp(self):
+        aug = augeas.Augeas()
+        ocsp_path = os.path.join('/files',
+                                 paths.HTTPD_NSS_CONF[1:],
+                                 'VirtualHost')
+        ocsp = aug.get(os.path.join(ocsp_path, 'directive[. = "NSSOCSP"]/arg'))
+
+        if ocsp is None:
+            aug.set(os.path.join(ocsp_path, 'directive[last()+1]'), 'NSSOCSP')
+
+        aug.set(os.path.join(ocsp_path, 'directive[. = "NSSOCSP"]/arg'), 'on')
+        aug.save()
```

Comment:
Ok, I'll chalk it up to a copy-paste problem on my end assuming others do an end-to-end test and confirm all three possible states (No starting NSSOCSP, NSSOCSP off, and NSSOCSP already on.